src/main/java/com/google/sps/data/TextAnalyser.java

```
@@ -0,0 +1,169 @@
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/*
+* This class creates objects that analyse text and 
+*/
+public final class TextAnalyser {
+  private final String message;
+
+  public TextAnalyser(String message) {
+     this.message = message.toLowerCase();
+  }
+
+  public float getSentimentScore() throws IOException {
+    try (LanguageServiceClient languageService = LanguageServiceClient.create()) {
+      Document doc =
+        Document.newBuilder().setContent(message).setType(Document.Type.PLAIN_TEXT).build();
+      Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
+        
+      return sentiment.getScore();
+    }
+  }
+
+  public String getMood() throws IOException {
+    float score = getSentimentScore();
+    
+    if(score == 1) {
+      return "very happy";
+    }
+ 
+    if(score == -1) {
+      return "so pessimistic";
+    }
+ 
+    int position = 0;
+    String[] moods = new String[]{"neutral", "calm", "relaxed", "serene", "contented",
+                                  "joyful", "happy", "delighted", "excited", "thrilled",
+                                  "tense", "nervous", "stressed", "upset", "sad", 
+                                  "depressed", "bored", "fatigued", "pessimisctic"};
+ 
+    score = score * 10;
+    position = (int) score;
+ 
+    if(position < 0) {
+      position = position * (-1) + 9;
+    }
+ 
+    return moods[position];
+  }
+
+  private ClassifyTextResponse classify() {
+    try (LanguageServiceClient language = LanguageServiceClient.create()) {
+      Document document = 
+            Document.newBuilder().setContent(message).setType(Type.PLAIN_TEXT).build();
+      ClassifyTextRequest request = ClassifyTextRequest.newBuilder().setDocument(document).build();
+        
+      return language.classifyText(request);
+    } catch(Exception e) {
```

If you catch all exceptions, log no error message, and return null to the caller, how are you going to figure out what went wrong when something goes wrong (as an engineer or a user)? In some way, all of the detailed information is discarded and at the same time you might have to do error checking via == null instead of using Java's exception mechanism.